anda/tools/pet/pet.spec

```diff
@@ -0,0 +1,32 @@
+Name:           Pet
+Version:        0.9.0
+Release:        1%?dist
+Summary:        Simple command-line snippet manager
+URL:            https://github.com/knqyf263/pet
+Source0:        %url/archive/refs/tags/v%version.tar.gz
+License:        MIT
+BuildRequires:  golang anda-srpm-macros
+Requires:       golang glibc
+Packager:       Owen Zimmerman <owen@fyralabs.com>
+ 
+%description
+%summary
+
+%prep
+%autosetup -n pet-%version
+ 
+%build
+%global debug_package %{nil}
+go build -gcflags "all=-N -l" -ldflags "-linkmode=external -extldflags=-Wl,--build-id" -o pet
```

Comment:
Why?

Reply:
Had a debug package issue

`
mock    | DEBUG: Processing files: Pet-debugsource-0.9.0-1.fc40.x86_64
mock    | DEBUG: RPM build errors:
mock    | DEBUG: error: Empty %files file /builddir/build/BUILD/pet-0.9.0/debugsourcefiles.list
mock    | DEBUG:     Empty %files file /builddir/build/BUILD/pet-0.9.0/debugsourcefiles.list
mock    | DEBUG: Child return code was: 1
`

As for the build ID workaround, that was set early on when I needed it, thought it was still relevant but I guess it builds just fine without it now, will remove it

Reply:
just do %define debug_package %nil